pontoon/base/static/js/fluent_interface.js

```diff
@@ -367,14 +392,19 @@ var Pontoon = (function (my) {
         }
       }
       else if (element.type === 'Placeable') {
-        if (element.expression.type === 'ExternalArgument') {
+        if (element.expression.type === 'VariableReference') {
           if (markPlaceables) {
             startMarker = '<mark class="placeable" title="External Argument">';
             endMarker = '</mark>';
           }
           string += startMarker + '{$' + element.expression.id.name + '}' + endMarker;
         }
-        else if (element.expression.type === 'MessageReference') {
+        else if (
+          [
+            'MessageReference',
+            'TermReference',
+          ].indexOf(element.expression.type) >= 0
```

Comment:
Nit: using `||` makes the intent clearer IMO:

    else if (element.expression.type === "MessageReference"
        || element.expression.type === "TermReference") {